pages/linux/unshadow.md

```diff
@@ -0,0 +1,11 @@
+# unshadow
+
+> Utility provided by the package "john", to combine /etc/passwd and /etc/shadow into a file that can be processed by John the Ripper. More information: <https://www.openwall.com/john/>.
```

Comment:
```suggestion
> Utility provided by John the Ripper project to obtain the traditional Unix password file if the system uses shadow passwords. More information: <https://www.openwall.com/john/>.
```

Reply:
Perhaps the `More information` could be on the next line, like this @andrik?
```suggestion
> Utility provided by the John the Ripper project to obtain the traditional Unix password file if the system uses shadow passwords.
> More information: <https://www.openwall.com/john/>.
```